libr/anal/p/anal_x86_cs.c

```diff
@@ -908,17 +908,35 @@ static void anop_esil(RAnal *a, RAnalOp *op, ut64 addr, const ut8 *buf, int len,
 	case X86_INS_CMPSB:
 	case X86_INS_CMPSS:
 	case X86_INS_TEST:
+    {
+        const char *opname;
+        switch (INSOP(0).size) {
```

Comment:
use tabs, cases go aligned at the same column as the switch and missing space before (